### PR TITLE
Script to evaluate easi models on a given mesh

### PR DESCRIPTION
**Pull request by sebwolf-de**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -17,5 +17,5 @@
 	path = submodules/ImpalaJIT
 	url = ../../Manuel1605/ImpalaJIT.git
 [submodule "place_receivers/submodules/PUML"]
-	path = place_receivers/submodules/PUML
+	path = submodules/PUML
```
Comment (Thomas-Ulrich) [inline .gitmodules:11]: I guess we could remove yaml-cpp, easi and ImpalaJIT from the submodules

File: evaluate_material/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+## Usage
+
+```
+./evaluate_easi -m mesh.h5 -e easi.yaml -o material
+```
+
+## What happens?
+
+1. Compute barycenter for each tetrahedron in `mesh.h5`.
+2. Evaluate all parameters, which are described in `easi.yaml` at the element barycenters.
```
Comment (Thomas-Ulrich) [inline evaluate_material/README.md]: `easi.yaml`, at

File: evaluate_material/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+make
+```
+
+## Usage
+
+```
+./evaluate_easi -m mesh.h5 -e easi.yaml -o material
+```
+
+## What happens?
```
Comment (Thomas-Ulrich) [inline evaluate_material/README.md]: What about "Description" ?

File: evaluate_material/src/main.cpp [hunk 4]
```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <string>
+
+#include <mpi.h>
+#include <utils/args.h>
```
Comment (Thomas-Ulrich) [inline evaluate_material/src/main.cpp]: include "utils/args.h"
same for logger.h ?

**Comment by Thomas-Ulrich** [comment]
Tested with a 3d velocity model of New Zealand: it's working!
image

**Comment by sebwolf-de** [comment]
> Tested with a 3d velocity model of New Zealand: it's working! image

Is this an approval?

**Comment by Thomas-Ulrich** [comment]
yes, once the comments are above are addressed ;-)

**Comment by sebwolf-de** [comment]
Strange, I don't see any comments.

File: evaluate_material/src/writer.h [hunk 5]
```diff
@@ -0,0 +1,44 @@
+#ifndef WRITER_H_
+#define WRITER_H_
+
+#include <cassert>
+#include <fstream>
+#include <iostream>
+
+#include "hdf5.h"
```
Comment (Thomas-Ulrich) [inline evaluate_material/src/writer.h]: <hdf5.h>?

File: evaluate_material/src/parameterDB.h [hunk 6]
```diff
@@ -0,0 +1,22 @@
+#ifndef PARAMETER_DB_H_
+#define PARAMETER_DB_H_
+
+#include <string>
+
+#include "easi/Component.h"
```
Comment (Thomas-Ulrich) [inline evaluate_material/src/parameterDB.h]: <easi/Component.h>

File: evaluate_material/README.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+## Usage
+
+```
+./evaluate_easi -m mesh.h5 -e easi.yaml -o material
+```
+
+## Description?
+
+1. Compute barycenter for each tetrahedron in `mesh.h5`.
+2. Evaluate all parameters, which are described in `easi.yaml`,b at the element barycenters.
```
Comment (Thomas-Ulrich) [inline evaluate_material/README.md]: typo "b" (added in last commit).

File: evaluate_material/src/parameterDB.cpp [hunk 8]
```diff
@@ -0,0 +1,42 @@
+#include "parameterDB.h"
+
+#include "easi/Component.h"
```
Comment (Thomas-Ulrich) [inline evaluate_material/src/parameterDB.cpp]: I don't know if that really important by <easi/Component.h>
(and other includes below).
I thought you would check them all.

**Review by Thomas-Ulrich — APPROVED** [review]
please address last comments

**Pull request merged**